README.md

```diff
@@ -1,75 +1,67 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Awesome Books
 
-> One paragraph statement about the project.
+> Book Store using JavaScript
 
 ![screenshot](./app_screenshot.png)
 
 Additional description about the project and its features.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML/CSS
+- JavaScript
+- Live Server
 
 ## Live Demo
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://bereketretta.github.io/Awesome_Books/)
 
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
 
-### Setup
-
-### Install
+git clone repo: `git clone https://github.com/BereketRetta/Awesome_Books.git`
 
-### Usage
+then `cd Capstone-project`
 
-### Run tests
+### Install
 
-### Deployment
+run `npm install` to install dependencies
 
 
 
 ## Authors
```

Comment:
- [ ] Please add other authors' social media links too, in order to introduce contributors to the future users of your project.
- [ ] Please make sure every project contributors make commits for this project